wandb/sdk/launch/deploys/kubernetes/launch-agent.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: launch-agent
+  namespace: wandb
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: launch-agent
+  template:
+    metadata:
+      labels:
+        app: launch-agent
+    spec:
+      serviceAccountName: wandb-launch-serviceaccount
+      containers:
+      - name: launch-agent
+        image: kaniko-build-test:v1
+        resources:
+          limits:
+            memory: "2Gi"
```

Comment:
Should these types of hardware limits be piped in through some kind of user setting? Seems like something the user would want to modify

Reply:
When users deploy the launch agent they'll have access to this file. Though I could put a comment indicating that they can modify this. And also a README in this directory pointing users to the line that matter.

As of now the launch agent in the kubernetes case will generally not run jobs locally. (Or at least that would be a very unexpected use case) we would push users to run the jobs using the kubernetes runner which will have its own resource requirements. So maybe 2 Gigs is just enough for the launch agent.... I don't know if there would be a way to go above that unless we have a memory leak somewhere. (Which is possible)